/builds/psumbera/mozilla-central-build/media/ffvpx/libavcodec/parsers.c:114:30: error: initialization of ?const FFCodecParser *? from incompatible pointer type ?const AVCodecParser *? [-Wincompatible-pointer-types]
  114 |     const FFCodecParser *p = parser_list[i];
      |                              ^~~~~~~~~~~

--- firefox-150.0/media/ffvpx/libavcodec/parser_list.c.old	2026-04-17 15:38:32.973192149 -0400
+++ firefox-150.0/media/ffvpx/libavcodec/parser_list.c	2026-04-17 15:39:25.091646612 -0400
@@ -1,6 +1,6 @@
 #include "config_components.h"
 
-static const AVCodecParser * const parser_list[] = {
+static const FFCodecParser * const parser_list[] = {
 #if CONFIG_VP8_PARSER
     &ff_vp8_parser,
 #endif
